packages/documentation-spa/docs/onboarding-guide/onboarding-guide.md

```diff
@@ -0,0 +1,283 @@
+
+
+## **OVERVIEW**
+ One Platform is a single place solution for teams within Red Hat to deploy their applications, websites and Microservices. The Stakeholders can easily deploy their applications without even being bothered about building and managing the Infrastructure hussles.
+
+
+## **ARCHITECTURE**
+1. _Relationship between users, developers, and platform_
+2. _Microservices, the core components, and integrations with internal tools and services_ 
+* SPAship is deploying each SPA and microservice in the IT’s Managed Platform. 
+* This lets the platform auto scale and the core services are able to call each other without being aware of where they are running. 
+
+
+
+## **CORE SERVICES**
+Along with Helper Functions that are directly injected in the browser window object when the OpNav is inserted into the DOM. So any SPA which has the OpNav included can use the provided features>
+
+
+### **Authentication**
+One platform has different strategies for authentication. At the Client-side, internal auth is supported (auth.redhat.com) and at the API gateway, JWT token from Internal auth. The API key is supported. SPAs are authenticated by default through SSI authentication support.
+
+**Tools/Integrations:** [auth.redhat.com](https://auth.redhat.com/)
+
+**OpAuthHelper** A Javascript object that provides some methods for getting the JWT Token, and   User Info.
+
+**Available methods**
+* onLogin(cb: (user) => void): void \
+Calls the callback function when the user is successfully logged in. Can be used to check when the user has logged in, to handle any API calls that require authentication.
+* isAuthenticated: boolean \
+Can be used to check if the user is logged in or not.
+* getUserInfo(): User | null \
+Returns the user details if the user is logged in, else returns null.
+
+
+### **SSI Components**
+
+One Platform provides global components, including a Navigation bar, Feedback action button, etc. These are provided as pluggable Apache Server Side Includes (SSI) snippets, with flexibility for the developers to customize and use it with their own applications. 
+
+**Tools/Integrations:** Web-components, Internal SSO for Authentication
+
+**Using ssi-templates for local development with other apps**
+
+1. Copy the `.env.example` file and rename it as `.env`. Add the values for the environment variables as expected.
+
+    ```
+    # Make a copy of the secrets.example.json
+    mv ./ssi-service/.env.example ./ssi-service/.env
+
+    # Edit the secrets file
+    nano ./ssi-service/.env
+    ```
+
+
+2. Add the following import in the your html files (preferably in the `<body>` tag before any other application javascript code/script tags)
+
+    ```
+    <body>
+      ...
+      <!-- SSI Include Snippet -->
+      <!--#include virtual="/.include/nav/default.html" -->
```

Comment:
This approach will become outdated soon

Reply:
@riginoommen  noted we would need the new opc-base and opc-nav